python/src/aiconfig/editor/client/src/components/SettingsPropertyRenderer.tsx

```diff
@@ -41,7 +41,7 @@ export function PropertyLabel(props: {
     <Group spacing="xs">
       <Text size="md">{propertyName}</Text>
 
-      <Tooltip label={propertyDescription} multiline>
+      <Tooltip label={propertyDescription} multiline maw={400}>
```

Comment:
What is `maw`?

Reply:
Alright big player I see you: https://github.com/lastmile-ai/aiconfig/blob/3dcf70bcddda7f7041c3bfb6e5d4b6c9a7abd4ab/python/src/aiconfig/editor/client/src/components/SettingsPropertyRenderer.tsx#L64

Reply:
It's `maxWidth` style setting for most mantine components. They expose some basic style modifiers (margin, width, etc.) on most components so you can modify their styles inline without needing classes/css.